### PR TITLE
adds custom slides to obs overlay, fixes #296

### LINKED ISSUE #296: custom slides dont show on bani overlay
- [x] blank slide
- [x] vaheguru slide
- [x] dhan guru _____ slide
- [x] custom announcements

### PR DESCRIPTION
**Pull request by inderpreetsingh**
screen shot 2018-09-04 at 7 06 37 pm

fixes issue #296

### CODE REVIEW

File: www/obs/index.html [hunk 1]
```diff
@@ -76,6 +79,7 @@
     };
 
     socket.on('show-line', function(request) {
+      console.log(request);
```
Comment (tsingh777) [inline www/obs/index.html]: remove?

File: www/js/controller.js [hunk 2]
```diff
@@ -472,10 +472,10 @@ module.exports = {
   },
 
   sendText(text, isGurmukhi) {
+    global.webview.send('show-empty-slide', { broadcast: false });
     global.webview.send('show-text', { text, isGurmukhi });
-    global.webview.send('show-empty-slide');
+    global.platform.ipc.send('show-empty-slide', { broadcast: false });
```
Comment (tsingh777) [inline www/js/controller.js]: what does the `{ broadcast: false }` argument do?

File: www/obs/index.html (hunk 1 above)
Comment (inderpreetsingh) [reply www/obs/index.html]: Gosh!

**Comment by tarunsingh5** [comment]
@inderpreetsingh can you address @tsingh777's comments

**Pull request merged**